packages/language/src/document.js

```diff
@@ -196,13 +197,34 @@ Document.PART_OF_SPEECH = {
  *     text.
  * @param {object[]} callback.annotation.tokens - Parts of speech that were
  *     detected from the text.
- * @param {object[]} callback.annotation.tokens.text - The piece of text
+ * @param {string} callback.annotation.tokens[].text - The piece of text
```

Comment:
For reference, Translate actually does this as well.

http://googlecloudplatform.github.io/google-cloud-node/#/docs/google-cloud/0.43.0/translate?method=detect